ansible/inventory

```diff
@@ -1,5 +1,90 @@
-[collectors]
-vps770.greenhost.nl
-[probes]
-#211.149.218.16
-87.107.121.133
+[gh:children] # group to gather new-style ACL
+wdc # technically it's i95.net, Radio Free Asia network, but GH has some boxes there
+hkg
+ams
+ath
+ber
+tpo
+bigv
+
+[wdc]
+a.echo.th.ooni.io
+c.web-connectivity.th.ooni.io
+munin.ooni.io
+b.collector.ooni.io
+
+# The machines below maybe can be discontinued
+ooni-proxy-dc.infra.ooni.io
+ooni-backend-testing.infra.ooni.io
+oonibackend-stress-test.infra.ooni.io
+
+tipitpitap.infra.ooni.io # Art-NO-ACCESS - no idea what this is
+ooni-fdroid-build.infra.ooni.io # Art-NO-ACCESS
+apk-build-test.infra.ooni.io # Art-NO-ACCESS - no idea what this is
```

Comment:
This are still unclaimed. We maybe want to move some of them to the delete section.